### DISCUSSION TITLE
possible to connect Etsy/walmart store through API?

### DISCUSSION THREAD

**Discussion by james400studio** [Sellbrite/Sellbrite-API#27] [Q&A]
Hey, we are starting a print on demand company and would like to publish new products on stores of our customers. Is it possible to all them to connect there eTSY/Walmart store directly from our website through API?

**Comment by rengler-godaddy** [comment]
Hello, thanks for reaching out! 

There isn’t a way to connect sales channels through the API. This can only be done directly inside the Sellbrite dashboard. 

Let us know if you have any other questions on this.
